visual-tests/tests/.empty-test-template.js

```diff
@@ -0,0 +1,25 @@
+import { test, expect } from '@playwright/test';
+import path from 'path';
+import { helpers } from '../imports/helpers';
+
+const stylesToAdd = [
+  helpers.cssFiles.cookieInfo,
+  helpers.cssFiles.dynamicDataFreeze
+];
+
+test(helpers.testTitle(path.basename(__filename)), async({ page }, workerInfo) => {
+  helpers.init(workerInfo);
+  await page.goto(helpers.testURL);
+  await expect(page).toHaveTitle(helpers.expectedPageTitle);
+  stylesToAdd.forEach(item => page.addStyleTag({ path: helpers.cssPath(item) }));
+
+  const table = page.locator(helpers.selectors.mainTable);
+
+  await table.waitFor();
+  helpers.mainTableFirstColumn = table.locator(helpers.selectors.mainTableFirstColumn);
+  helpers.tbody = table.locator(helpers.selectors.mainTableBody);
+  helpers.thead = table.locator(helpers.selectors.mainTableHead);
```

Comment:
```suggestion
  await table.waitFor();
  
  const mainTableFirstColumn = table.locator(helpers.selectors.mainTableFirstColumn);
  const tbody = table.locator(helpers.selectors.mainTableBody);
  const thead = table.locator(helpers.selectors.mainTableHead);
```
I think that it's cleaner to just assign the elements to the local variables and use them within the test instead of assigning them to helpers utils.

Reply:
I agree, I did it for only one reason - I wanted to keep them always declared and do not force testers to declare them manually, but then there is a chance that eslint will be yelling about unused vars.

Will `/* eslint-disable no-unused-vars */` be satisfying in this case?

Reply:
For the test template, I'd propose commenting 3 lines as starter/hint for the writer then.

Reply:
The same thing appears in few files - it was done intentionally as a compromise to make tester's life easier, now should be modified as we discussed.